website/src/pages/log-streaming/content-overview/index.mdx

```diff
@@ -1,46 +1,46 @@
 ---
-Title: Event Streaming
-Description: How does Event Streaming work & how to leverage it
+Title: Log Streaming
+Description: How does Log Streaming work & how to leverage it
 Keywords: 'ibm cloud'
 ---
 
 <PageDescription>
 
-What is Event Streaming?
+What is Log Streaming?
 
 </PageDescription>
 
 
-Event Streaming enables IBM Log Analysis to produce content to a message bus queue and Topic. IBM Log Analysis streaming helps you to connect third party consumers of topics to ingest into dashboards for visualization of event log data. 
+Log Streaming enables IBM Log Analysis to produce content to a message bus queue and Topic. IBM Log Analysis streaming helps you to connect third party consumers of topics to ingest into dashboards for visualization of event log data. 
```

Comment:
Revise this to be inclusive of both IBM Log Analysis and Activity Tracker services. It's ok if in the rest of the example you refer to IBM Log Analysis, just need to make it clear that both work the same way to stream data to an Event Streams instance